### PR TITLE
Allow an author to edit node metadata, including child order

### PR DESCRIPTION
**Pull request by tessereth**
**DO NOT MERGE UNTIL AFTER #180**

Obviously this is not the best user experience possible but it makes it possible to change node order.

screen shot 2016-07-12 at 4 39 15 pm

### CODE REVIEW

**Comment by micapam** [comment]
What if one were to set all order values the same? 3, 3, 3? I'm assuming that passes validation & just results in a random order - correct?

**Comment by tessereth** [comment]
Correct. Nodes with the same number will come out in database order.

**Comment by micapam** [comment]
In that case, LGTM

**Comment by ltankey** [comment]
LGTM on the proviso that the DB query for siblings with the same weight returns the same random order every time.

File: app/controllers/editorial/nodes_controller.rb [hunk 1]
```diff
@@ -101,8 +96,7 @@ def configure_defaults!
 
     def try_save
       return false unless @form.validate(params.require(:node).permit!)
```
Comment (ltankey) [inline app/controllers/editorial/nodes_controller.rb]: Calling `@form.validate` will return a boolean expression with the result of the validation, so this method is probably unnecessary.

**Comment by ltankey** [comment]
Just a final thought - should we provide the ability to add this metadata on node/page creation?

**Comment by tessereth** [comment]
We currently allow setting the parent on node/page creation. But when you create a page it doesn't have any children. Possibly you could set where a page sits among it's siblings but that's a little harder (requires updating all the other siblings). It's probably a useful feature to have eventually though.

**Pull request merged**